packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -6,7 +6,9 @@ import { storeSetting, getSetting } from '../../utils/settings';
 import TourGuide from './tour-guide';
 import { getImageLocation } from '../../public-path';
 
-type TJoyrideConfig = Record<'showProgress' | 'spotlightClicks' | 'disableBeacon' | 'disableOverlay', boolean>;
+type TJoyrideConfig = Record<'showProgress' | 'spotlightClicks' | 'disableBeacon', boolean>;
```

Comment:
You can `pick` from `TStep` instead of creating a record based on static items

Reply:
done ✅